ostorlab_insecure_flutter_app/lib/bugs/biometric_none_cryptobject.dart

```diff
@@ -3,18 +3,17 @@ import 'package:local_auth/local_auth.dart';
 import 'package:ostorlab_insecure_flutter_app/bug_rule.dart';
 
 class BiometricNoneCryptObject extends BugRule {
-
   static const String _tag = 'BiometricNoneCryptObject';
 
-  String get description => 'Use Biometric authentication without a crypto Object.';
+  String get description =>
+      'Use Biometric authentication without a crypto Object.';
 
-  Future<void> run() async {
+  Future<void> run(String user_input) async {
```

Comment:
Please rename to input.

Reply:
Applies to all.

Reply:
[2c3d87d](https://github.com/Ostorlab/ostorlab_insecure_android_app/pull/22/commits/2c3d87d9f679ffa607a88cd4cc55780e39e06ece)